conda_build_config.yaml

```diff
@@ -1,6 +1,7 @@
 mpi:
```

Comment:
Please don't edit any files outside of your recipe directory.  Instead add a new `conda_build_config.yaml` in your recipe dir for that recipe